src/js/components/atoms/buttons/buttons.scss

```diff
@@ -25,7 +25,7 @@ input[type="submit"] {
         margin-bottom: $buildingUnit;
     }
 
```

Comment:
as far as i know, `font-family: inherit`, is already the default, so maybe this property can be removed?